### DISCUSSION TITLE
Фильтр по полю "дерево"

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#291] [Как это сделать?] [answered]
Подскажите пожалуйста, как правильно настраивается фильтр, если поле по которому мы ищем является select tree. Например я хочу выбрать родительскую категорию, чтобы все входящие подкатегории были включены в поиск.

![Screenshot_221](https://user-images.githubusercontent.com/113771444/207798831-613101e7-61e8-48f6-910a-441131b23ebf.jpg)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
В этом случае у вас должен быть Множественное включено, что и видно на этой картинке и дальше мне кажется это делает вот эта рука в форме ящерицы 🦎

**Reply by mihail-cornev** [reply]
Спасибо, не догадался нажать на ящерицу. Оказывается чтобы выбрать -под-под категории, надо несколько раз нажать на ящерицу. Не мог врубиться сразу... как-то не сильно удобно.
